pkg/printers/internalversion/printers_test.go

```diff
@@ -828,12 +828,30 @@ func TestPrintNodeStatus(t *testing.T) {
 		{
 			node: api.Node{
 				ObjectMeta: metav1.ObjectMeta{
+					Name:   "foo10",
+					Labels: map[string]string{"node-role.kubernetes.io/master": "", "kubernetes.io/role": "node", "kubeadm.alpha.kubernetes.io/role": "node"},
```

Comment:
You can have empty labels, and the value doesn't matter here ("reserved for future usage" perhaps?)